torchok/tasks/self_supervised/swav.py

```diff
@@ -0,0 +1,100 @@
+from typing import Dict, List
+from omegaconf import DictConfig
+from torch import Tensor, nn
+from lightly.models.modules import SwaVPrototypes
+
+from torchok.constructor import TASKS
+from torchok.tasks.classification import ClassificationTask
+
+
+@TASKS.register_class
+class SwaVTask(ClassificationTask):
```

Comment:
Can you add an example config for this task?

Reply:
I will add later when i get the same results as lightly

Reply:
Added